extras/python/sin1024_int8.py

```diff
@@ -1,40 +0,0 @@
-## generates a sin-shaped table with values -128 to 127
```

Comment:
Accidentally removed while splitting out the Oscil16-changes?

Reply:
whoops